From: Simon McVittie <smcv@collabora.com>
Date: Thu, 5 Apr 2018 11:19:47 +0100
Subject: Spelling: fix "every time"

Detected by Debian's Lintian tool.

Also fix the spelling of "linking" in one error message.

Signed-off-by: Simon McVittie <smcv@collabora.com>
Forwarded: GNOME/glib!354
Applied-upstream: 2.59.0, commit:214c991d5b3a10368684142a706bc8bc826342b3
---
 configure.ac                            | 2 +-
 docs/reference/gobject/glib-mkenums.xml | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index f6797a5..93a8d4c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1853,7 +1853,7 @@ FUNC_NO_LOCALTIME_R="the 'g_date_set_time' function will not be MT-safe
 
 AIX_COMPILE_INFO="AIX's C compiler needs to be called by a different name, when
 		linking threaded applications. As GLib cannot do that 
-		automatically, you will get an linkg error everytime you are 
+		automatically, you will get an linking error every time you are
 		not using the right compiler. In that case you have to relink 
 		with the right compiler. Ususally just '_r' is appended 
 		to the compiler name."
diff --git a/docs/reference/gobject/glib-mkenums.xml b/docs/reference/gobject/glib-mkenums.xml
index a59cacf..7e0eabd 100644
--- a/docs/reference/gobject/glib-mkenums.xml
+++ b/docs/reference/gobject/glib-mkenums.xml
@@ -253,7 +253,7 @@ will be appended to the template's <literal>file-tail</literal> section.
 <varlistentry>
 <term><option>--eprod</option> <replaceable>TEXT</replaceable></term>
 <listitem><para>
-Emits <replaceable>TEXT</replaceable> everytime an enum is encountered
+Emits <replaceable>TEXT</replaceable> every time an enum is encountered
 in the input files.
 </para></listitem>
 </varlistentry>
